hsG3DDeviceSelector support for OpenGL (#1077)
H-uru/Plasma: Sources/Plasma/FeatureLib/pfGLPipeline/CMakeLists.txt

        plPipeline
    PRIVATE
        plStatusLog
        epoxy::epoxy
    INTERFACE
        pnFactory
)

> if(APPLE AND CMAKE_SYSTEM_NAME MATCHES "Darwin")
>     # On macOS, we need to link against OpenGL.framework for access to the CGL
>     # functions. On other platforms, the OpenGL library is loaded dynamically
>     # by epoxy.
>     target_link_libraries(pfGLPipeline PRIVATE "-framework OpenGL")
> endif()
> 

Reviewer: ```suggestion
# On macOS, we need to link against OpenGL.framework for access to the CGL
# functions. On other platforms, the OpenGL library is loaded dynamically
# by epoxy.
target_link_libraries(pfGLPipeline PRIVATE "$<$<PLATFORM_ID:Darwin>:-framework OpenGL>")
```
